tests/integration.rs

```diff
@@ -4996,7 +4996,6 @@ mod common_parallel {
         handle_child_output(r, &output);
     }
 
-    #[allow(clippy::useless_conversion)]
```

Comment:
The commit title should have been "tests: Remove "useless_conversion" clippy suppression"